### PR TITLE
Pdct 1285/skeleton endpoint for ingest

### PR DESCRIPTION
**Pull request by annaCPR**
# Description
- adds a skeleton endpoint for ingesting json data

Please include:

- a summary of the changes
- links to any related issue/ticket
- any additional relevant motivation and context
- details of any dependency updates that are required for this change

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [ ] Patch
- [x] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: tests/unit_tests/routers/ingest/test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import requests
```
Comment (diversemix) [inline tests/unit_tests/routers/ingest/test.py:1]: Did this file get added by mistake?

**Review by diversemix — APPROVED** [review]
All looks good - just a question about the `test.py` file

File: app/service/corpus.py [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
+import app.clients.db.session as db_session
 from app.errors import RepositoryError, ValidationError
 from app.repository import corpus_repo
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def get_corpus_org_id(db: Session, corpus_import_id: str) -> int:
+def get_corpus_org_id(corpus_import_id: str, db: Optional[Session] = None) -> int:
```
Comment (diversemix) [inline app/service/corpus.py:13]: I'm intrigued why this has changed?

Comment (annaCPR) [reply app/service/corpus.py:13]: This follows the pattern we have in all the other services. Before this change, the corpus service was only used in other services where the db session was passed in. I needed to use this in my ingest router and didn't want to add a dependency to the db. So I made the db parameter optional (like in other services) and it gets initialised if it is not passed in.

Comment (diversemix) [reply app/service/corpus.py:13]: Yup... that was a good refresher for me :) thanks

File: tests/unit_tests/routers/ingest/test.py (hunk 1 above)
Comment (diversemix) [reply tests/unit_tests/routers/ingest/test.py:1]: Question still stands :pray:

**Review by diversemix — APPROVED** [review]
looking great! Really good solid start on this - just the file `test.py` which I'm not sure if it was added by mistake?

File: tests/unit_tests/routers/ingest/test.py (hunk 1 above)
Comment (annaCPR) [reply tests/unit_tests/routers/ingest/test.py:1]: Yes, well spotted. I will remove now.

**Review by diversemix — DISMISSED** [review]
Great!

**Review by diversemix — APPROVED** [review]
neat!

**Pull request merged**